demo/src/lib.rs

```diff
@@ -80,6 +92,28 @@ impl RenderOptions {
             .build(ui, &mut blur_pass_count);
 
         self.blur_pass_count = blur_pass_count as usize;
+        let tonemapper_names = [
```

Comment:
can these strings be a constant array that's right next to the enum? This will make it more obvious when the enum is changed to also update the array to be the correct order/same size. (We could add a to_string() function to the enum that indexes into that array).

Or make to_string() match like:
```rust
match value {
     ToneMapperType::LogDerivative => "LogDerivative"
}
```

Actually a to_string() match function by itself (instead of an array) seems like a good way to do it. Very readable and least error-prone